02-express-tutorial/app.js

```diff
@@ -1 +1,104 @@
-console.log('Express Tutorial')
+const express = require("express");
+const path = require("path");
+
+const { products } = require("./data");
+
+const app = express();
+console.log(app);
+
+const PORT = 3000;
+
+app.use(express.static("./public"));
+
+app.get("/api/v1/test", (req, res) => {
+  res.json({ message: "It worked!" });
+});
+
+app.get("/api/v1/products", (req, res) => {
+  res.json(products);
+});
+
+app.get("/api/v1/products/:productID", (req, res) => {
+  // console.log(req.params);
```

Comment:
If the code is dead, then just remove it, there is no need to keep the code in a comment if it is not necessary anymore. If you think it might be relevant at some point, then do a commit with the comment and then remove it, so you can access it later.
However, you will probably never use it again, mostly dead code continues dead.